Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiSwitcher.cpp

```diff
@@ -134,8 +147,9 @@ namespace AzToolsFramework::ViewportUi::Internal
         if (auto buttonIt = AZStd::find_if(buttons.begin(), buttons.end(), found); buttonIt != buttons.end())
         {
             QString buttonName = ((*buttonIt)->m_name).c_str();
-            QIcon buttonIcon = QIcon(QString(((*buttonIt)->m_icon).c_str()));
-            m_activeButton->setIcon(buttonIcon);
+            auto buttonPixmap = CreateIconMask((*buttonIt)->m_icon);
```

Comment:
question: Should we disable/remove this for now as it's not working 100% correctly?